gips/utils.py

```diff
@@ -389,6 +449,44 @@ def mosaic(images, outfile, vector):
     return crop2vector(imgout, vector)
 
 
+def gridded_mosaic(images, outfile, rastermask, interpolation=0):
+    """ Mosaic multiple files to grid and mask specified in rastermask """
```

Comment:
Optional:  Time permitting, do you think this would be cleaner?  Pass in `filenames` and `images[0]` instead of `images`, then it seems like a bunch of lines simplify or even go away entirely.